pages/android-walkthrough.md

```diff
@@ -0,0 +1,94 @@
+#Walkthrough: Add a Tangram map to an Android application
+
+With the Tangram Android SDK, you can quickly add beautiful and useful 2D and 3D maps to an Android application. Tangram uses customizable scenes that enable fine control over almost every aspect of your map's appearance, including symbols, lighting, geometry, and feature labels. The Tangram Android SDK is [open source](https://github.com/tangrams/tangram-es), and supports several vector data formats.
+
+Follow this guide to add a Tangram map to your Android application. If you're looking for a simple way to add a pre-configured map and related geographic services to your application, you should check out the [Mapzen Android SDK](https://mapzen.com/documentation/android/).
+
+The Tangram Android SDK can be used with any Android build system that supports Gradle. For this guide we will use [Android Studio](http://developer.android.com/sdk/index.html).
+
+> This guide will assume some knowledge of Android software development. If you want an introduction to writing Android applications, visit the [Android developer site](http://developer.android.com/training/index.html).
+
+###Run the Simple Map demo application
+
+First, let's look at a sample application that demonstrates simple usage of a Tangram map.
+
+Start by downloading the repository of Tangram Android demos. You can do this in a terminal window with the following command:
+
+```
+git clone --recursive https://github.com/tangrams/tangram-android-demos.git
+```
+
+Next, import the project by opening Android Studio, choosing "Import project", and selecting the "tangram-android-demos" folder.
+
+With the project open, choose `simplemap` from the run menu and run the demo on a device or an emulator. Your device will show a map that responds to dragging, pinching, and rotating gestures.
+
+Finally, open the `simplemap` module in Android Studio to see the code used for the demo. In the next section, we will look at all of the steps needed to add a map like this to your application.
+
+###Steps to add Tangram to an Android application
+
+1. **Add Tangram as a Gradle dependency.**
+
+  In the `build.gradle` file for your application's module, add the following dependency:
+
+  ```
+  compile 'com.mapzen.tangram:tangram:0.2-SNAPSHOT'
```

Comment:
We would want to use a tangram release version here right, instead of the snapshot?